publishable/lang/el/generic.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+return [
+    'is_rtl'                 => 'όχι',
```

Comment:
This field is used to check if language is rtl, so even if it's still going to work as far as is not literal `'true'` it's probably better if it's set as `'false'`.

Reply:
Fixed [here](https://github.com/the-control-group/voyager/commit/aa44a8e40c72d4282e31f2df9637291c8dc069e1)